src/lib/one-graph/cli-client.js

```diff
@@ -51,13 +57,31 @@ InternalConsole.registerConsole(internalConsole)
  * @param {function} input.onEvents A function to call when CLI events are received and need to be processed
  * @param {string} input.sessionId The session id to monitor CLI events for
  * @param {StateConfig} input.state A function to call to set/get the current state of the local Netlify project
+ * @param {any} input.site The site object
  * @returns
  */
 const monitorCLISessionEvents = (input) => {
-  const { appId, netlifyGraphConfig, netlifyToken, onClose, onError, onEvents, sessionId, state } = input
+  const { appId, netlifyGraphConfig, netlifyToken, onClose, onError, onEvents, sessionId, site, state } = input
 
   const frequency = 5000
+  // 30 minutes
+  const defaultHeartbeatFrequency = 1_800_000
   let shouldClose = false
+  let nextMarkActiveHeartbeat = defaultHeartbeatFrequency
+
+  const markActiveHelper = async () => {
+    const fullSession = await OneGraphClient.fetchCliSession({ authToken: netlifyToken, appId, sessionId })
+    // @ts-ignore
+    const heartbeatIntervalms = fullSession.session.cliHeartbeatIntervalMs || defaultHeartbeatFrequency
+    nextMarkActiveHeartbeat = heartbeatIntervalms
```

Comment:
[sand] The above two lines can be merged by removing the intermediary `heartbeatIntervalms` variable.

Reply:
I specifically pulled it out so that it's obvious what the intention of that variable is via its name.

Reply:
Alright :+1: